tad/models/status.py

```diff
@@ -0,0 +1,7 @@
+from sqlmodel import Field, SQLModel  # type: ignore
+
+
+class Status(SQLModel, table=True):
+    id: int = Field(default=None, primary_key=True)
+    name: str
+    sort_order: float
```

Comment:
Why is the sort_order a float and not an integer?

Reply:
Because sort order is calculated depending on the siblings (f.e. between sort_order = 1, sort_order=2, the new sort_order = 1,5).